config/params/loader_test.go

```diff
@@ -130,6 +131,7 @@ func TestModifiedE2E(t *testing.T) {
 	c.TerminalTotalDifficulty = "0"
 	c.AltairForkEpoch = 0
 	c.BellatrixForkEpoch = 0
+	c.DenebForkEpoch = 358
```

Comment:
Not sure what to put here. deneb-integration was quite different. 
See:

https://github.com/prysmaticlabs/prysm/blob/deneb-integration/config/params/loader_test.go#L132-L135